subprojects/execution/src/test/groovy/org/gradle/internal/execution/history/impl/FileCollectionFingerprintSerializerTest.groovy

```diff
@@ -43,7 +44,7 @@ class FileCollectionFingerprintSerializerTest extends SerializerSpec {
         def hash = HashCode.fromInt(1234)
 
         def rootHashes = ImmutableMultimap.of(
-                "/1", FileSystemLocationFingerprint.MISSING_FILE_SIGNATURE,
+            "/1", MissingFileSnapshot.SIGNATURE,
```

Comment:
Indent.

Reply:
Indeed, I've reviewed it from Idea and though it was some folding inconsistency :)
<img width="416" alt="Screen Shot 2019-06-28 at 15 46 33" src="https://user-images.githubusercontent.com/1841944/60343131-f6334500-99bb-11e9-972b-002e9c9b8c06.png">